plugins/mcreator-core/blockly/js/block_procedure.js

```diff
@@ -0,0 +1,119 @@
+const PROCEDURE_DEPENDENCIES_MUTATOR_MIXIN = {

```

Comment:
I think this file is unnecessary, the mixin + mutator can be in `mcreator_extensions.js` and the blocks in `mcreator_blocks.js`

Reply:
Not really, it's pretty standard to define the mixin and then the mutator. Also this isn't a helper function, as it doesn't return a mixin

Reply:
Not really, it's pretty standard to define the mixin and then the mutator. Also this isn't a helper function, as it doesn't return a mixin

Reply:
I mean replacing this mixin with a function that returns similar mixins (like what you're adding in your `any_of/all_of` predicates PR).

Reply:
I'm not sure what you mean

Reply:
I mean a function like this one:
https://github.com/MCreator/MCreator/blob/1ac4123337380cb6cb1dae0bd4b9919d05e6a794/plugins/mcreator-core/blockly/js/mcreator_extensions.js#L188
that returns mixins like `PROCEDURE_DEPENDENCIES_MUTATOR_MIXIN` but with specified input and mutator block names/types.

Reply:
Ok, then I don't see why it should be replaced with an helper function. You can just move the mixin+mutator

Reply:
✔️ Fixed